spisea/tests/test_synthetic.py

```diff
@@ -730,6 +731,355 @@ def match_model_masses2(isoMasses, starMasses):
     return
 
 
+<<<<<<< HEAD:popstar/tests/test_synthetic.py
+def test_phot_consistency(filt='all'):
```

Comment:
FYI: the test_phot_consistency function has been depreciated and is no longer part of the SPISEA code base.